File: app/src/main/java/com/przemolab/oknotifier/activities/SettingsActivity.java
package com.przemolab.oknotifier.activities;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.app.ActionBar;

import com.przemolab.oknotifier.R;

public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
        setupActionBar();
    }

    private void setupActionBar() {
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
        }
    }
}

File: app/src/main/java/com/przemolab/oknotifier/activities/SettingsActivity.kt
package com.przemolab.oknotifier.activities

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.app.ActionBar

import com.przemolab.oknotifier.R

class SettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)
        setupActionBar()
    }

    private fun setupActionBar() {
        val actionBar = supportActionBar
        actionBar?.setDisplayHomeAsUpEnabled(true)
    }
}
